### PR TITLE
Issues/4121 update pod and fonts

### LINKED ISSUE #4121: Update to v0.4.3 of the shared pod
The shared pod has been updated to version 0.4.3, updating the Merriweather fonts it provides. 

Update the app to use this version of the pod. 
Update the app to use Merriweather, instead of Merriweather Light for its content text in the reader, comments, posts, etc. 
Note

### PR DESCRIPTION
**Pull request by aerych**
Closes #4121 
Updates the shared pod and switches from Merriweather-Light to regular Merriweather for content views. 

To Review: 
Tour the app where we use the Merriweather fonts and look for the wrong font displayed.
- [x] My Sites ( post list, page list )
- [x] Reader (list, detail, comments)
- [x] Editor (visual editor)
- [x] Notifications (comments, details)

@jleandroperez Since this also updates fonts used in Notifications would you mind reviewing? 

Needs Review: @jleandroperez

### CODE REVIEW

File: WordPress/Classes/System/WordPressAppDelegate.m [hunk 1]
```diff
@@ -397,7 +397,13 @@ - (void)runStartupSequenceWithLaunchOptions:(NSDictionary *)launchOptions
     [[AFNetworkActivityIndicatorManager sharedManager] setEnabled:YES];
     self.userAgent = [[WPUserAgent alloc] init];
     [self setupSingleSignOn];
-    
+
+    // WORKAROUND: Preload the Merriweather regular font to ensure it is not overridden
+    // by any of the Merriweather varients.  Size is arbitrary.
+    // See: https://github.com/wordpress-mobile/WordPress-Shared-iOS/issues/79
+    // Remove this when #79 is resolved.
+    [WPFontManager merriweatherRegularFontOfSize:16.0];
```
Comment (jleandroperez) [inline WordPress/Classes/System/WordPressAppDelegate.m]: Ahhhh iOS!!!

**Comment by jleandroperez** [comment]
Looks great @aerych!

:shipit:

File: WordPress/Classes/System/WordPressAppDelegate.m (hunk 1 above)
Comment (aerych) [inline WordPress/Classes/System/WordPressAppDelegate.m]: ikr!

**Comment by aerych** [comment]
Thanks @jleandroperez!

**Pull request merged**